toolchain/build-toolchain.sh

```diff
@@ -1,69 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env sh
 
-MAKE_OPTIONS="-j1"
-
-function download() { # $1=url, $2=file
-	if [ ! -f $2 ]; then
-		curl "$1$2" > "$2"
-	fi
-
-	if [ ! -d "${2%.tar.xz}" ]; then
-		tar xf "$2"
-	fi
+download() {
+	wget "$1" -O "$2"
+	tar xvf "$2"
 }
 
-function download-binutils() {
-	url="ftp://ftp.gnu.org/gnu/binutils/"
-	file=$(curl -sl $url | grep -E 'binutils-[0-9.]+.tar.xz$' | sort -V | tail -n1)
-	download "$url" "$file"
-	echo "$PWD/${file%.tar.xz}"
+download_binutils() {
+	printf " - binutils\n"
+	version="2.34"
+	binutils="binutils-$version.tar.xz"
+	download "https://ftp.gnu.org/gnu/binutils/$binutils" "$binutils"
+	export binutils_src="$PWD/binutils-$version"
 }
 
-function download-gcc() {
-	url="ftp://ftp.gnu.org/gnu/gcc/"
-	folder=$(curl -sl $url | grep -E 'gcc-[0-9.]+$' | sort -V | tail -n1)
-	url="$url$folder/"
-	file="$folder.tar.xz"
-	download "$url" "$file"
-	echo "$PWD/$folder"
+download_gcc() {
+	printf " - gcc\n"
+	version="9.3.0"
```

Comment:
why not 10.1.0 ?

Reply:
I completely forgot that 10.1.0 existed. I'll change it in the next commit.